CSS/Drawing/css-pseudo-elements.md

```diff
@@ -0,0 +1,121 @@
+# CSS pseudo element
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[More about pseudo elements](https://www.html5rocks.com/en/tutorials/shapes/getting-started/)'
+
+---
+## Content
+
+*Pseudo elements* are used in CSS to style specified parts of an element.
+
+By using `::first-letter` pseudo element we can style the first letter of a text. For example we can style the first letter of all `<p>` elements:
+
+```
+p::first-letter{
+  color: #ff0000;
+  font-size: medium;
+}
+```
+
+We can use a similar property to style the first line of a text (`::first-line`):
+
+```
+p::first-line{
+  color: #ff0000;
+  font-variant: caps;
+}
+```
+
+Note that we are able to use pseudo elements with CSS classes as well:
```

Comment:
pseudo elements can be combined with any other CSS selector, here's an example with a class: